src/react-native/18-maps/maps.md

```diff
@@ -107,11 +149,42 @@ If you’ve implemented the solution correctly, the Map should be rendering in y
 
 ## Objective 2: Add restaurant pins with tooltips to map
 
-TODO
+<img alt="Screenshot of the a successfully implemented Google Map in the application." src="../static/img/react-native/18-maps/01-solution.png" style="max-height: 750px; border: 4px solid black; border-radius: 25px;"/>
+
+### Marker
+
+```tsx
+import MapView, { Marker, PROVIDER_GOOGLE } from "react-native-maps";
+
+function MapWithMarker({ initialRegion }) {
+  return (
+    <MapView
+      style={{ minHeight: "100%", height: 500 }}
+      provider={PROVIDER_GOOGLE}
+      initialRegion={initialRegion}
+    >
+      <Marker
+        coordinate={{
+          latitude: 37.78825,
+          longitude: -122.4324,
+        }}
+        onCalloutPress={() => {
+          console.log("Marker was pressed");
+        }}
+        title="Title of the Marker"
+        description="A brief description"
+      />
+    </MapView>
+  );
+}
+```
 
-### Concept TODO
+The `Marker` component is used to render a pin on the map. It takes several props, but the most important ones are:
 
-TODO
+- `coordinate`: The coordinates of the marker.
+- `onCalloutPress`: A function that is called when the callout view is pressed.
+- `title`: The title of the marker.
+- `description`: A description of the marker.
```

Comment:
What’s the difference between these two / where do they show up? Is the title the darker text and the description lighter text?